plugins/opa/pkg/opa/opa.go

```diff
@@ -67,7 +67,7 @@ func processAllChecks(ctx context.Context, checkInstances []CheckSetting, checks
 			}
 		}
 	}
-	return actionItems, lastError
+	return actionItems, nil
```

Comment:
@rbren I wanted to call out the reason I no longer return `lastErr` is to avoid the calling function and main.go propagating these errors as a panic.

Reply:
🤔 I think we should still pass the error back here, but not panic when it does return